.github/workflows/docker_all-modules.yml

```diff
@@ -0,0 +1,36 @@
+# This is a workflow to build and optionally push all module Docker images
+#
+# This workflow will run only on manual triggers
+
+name: Build all Docker images
+on:
+  workflow_dispatch:
+    inputs:
+      push-ecr:
+        description: "Push to AWS ECR"
+        type: boolean
+        required: false
```

Comment:
Should this also be scheduled?

Reply:
If it should be scheduled, we should probably figure out how we'll know about it if it fails!